src/App.js

```diff
@@ -66,6 +73,137 @@ class App extends Component {
     });
   }
 
+  getLeaderboardHead() {
+    const w = parseInt(window.innerWidth, 10);
+
+    if (w <= 768) {
+      return (
+        <tr>
+          <th scope="col">
+            <h4>Atletas</h4>
+          </th>
+        </tr>
+      );
+    } else if (w > 768 && w <= 1320) {
+      return (
+        <tr>
+          <th scope="col">Atletas</th>
+          <th scope="col">18.1</th>
+          <th scope="col">18.2</th>
+          <th scope="col">18.2a</th>
+          <th scope="col">18.3</th>
+          <th scope="col">18.4</th>
+          <th scope="col">18.5</th>
+        </tr>
+      );
+    } else {
+      return (
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Atletas</th>
+          <th scope="col">Pontos</th>
+          <th scope="col">18.1</th>
+          <th scope="col">18.2</th>
+          <th scope="col">18.2a</th>
+          <th scope="col">18.3</th>
+          <th scope="col">18.4</th>
+          <th scope="col">18.5</th>
+        </tr>
+      );
+    }
+  }
+
+  getScoreAsTable(scores) {
+    return scores.map((score, index) => {
+      return ( score.rank ?
```

Comment:
In this case, you are returning the conditional itself. You must check the conditional before this return and return just the html elements